core/templates/dev/head/pages/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -385,5 +385,10 @@ oppia.controller('SettingsTab', [
     $scope.closeRolesForm = function() {
       $scope.isRolesFormOpen = false;
     };
+
+    // Meathod to check if title is given
+    $scope.hasTitleNotBeenGiven = function() {
```

Comment:
```suggestion
    $scope.isTitlePresent = function() {
```
I think in general it is better to have function names that aren't negated. In case we use it later, !function() would be a double negation. It would be more readable to have a single negation :)

Reply:
Done, changed it.